daffodil-test/src/test/resources/edu/illinois/ncsa/daffodil/section11/content_framing_properties/ContentFramingProps.tdml

```diff
@@ -1000,7 +1000,20 @@
             <xs:complexType>
               <xs:sequence>
                 <xs:element name="string7Bit" type="xs:string" dfdl:length="1" dfdl:lengthKind="explicit" />
-                <xs:element name="string8Bit" type="xs:string" dfdl:length="1" dfdl:lengthKind="explicit" dfdl:encoding="US-ASCII" dfdl:alignment="8" />
+                
```

Comment:
Try this test without the sequence that forces 8-bit alignment. It should work now and do the aligning itself.

Reply:
There is this big comment in the commit message:

_There is an issue with test mixed_encodings2 in
ContentFramingProps.tdml.
<element A - a single 7 bit character - LSBF>
<element B - a UTF-8 character so has bitOrder MSBF, also specifies
alignment =8 bits.
Question is should the alignment=8 be enough - and should that perform
alignment using the LSBF that we need in order to get to the byte
boundary so we can change bitOrder?
Or does the alignment region for B have to be rendered in MSBF bit
order.

We want to align to 8 bit BEFORE we change the bitOrder. I.e., alignment
is really about wherever the prior term left us as far as bitOrder._

That needs to be explained here. Possibly tests created to make this clearer.

Reply:
Bug DAFFODIL-1888 created to capture this issue. Test case in TDML refers to this ticket.